src/sec_edgar_bulker/core/header_generator.py

```
@@ -0,0 +1,95 @@
+"""Header generator for SEC EDGAR requests."""
+
+import random
+import re
+from typing import Dict
+
+from .models import Config, HeaderGeneratorSettings, StaticHeaders
```

Ruff (0.8.2)

`.models.HeaderGeneratorSettings` imported but unused

Remove unused import

(F401)